python/pyspark/sql/udtf.py

```diff
@@ -376,10 +376,13 @@ def __call__(self, *args: "ColumnOrName", **kwargs: "ColumnOrName") -> "DataFram
         from pyspark.sql.classic.column import _to_java_column, _to_seq
 
         from pyspark.sql import DataFrame, SparkSession
+        from pyspark.sql.functions import lit
 
         spark = SparkSession._getActiveSessionOrCreate()
         sc = spark.sparkContext
 
+        args = tuple(lit(arg) for arg in args)
```

Comment:
Hmm we can support column references with lateral joins for Python UDTFs in the future, so the `arg` input here might not always be literals (e.g. `df.join(MyUDTF("c1", "c2"), ...)`  where "c1" and "c2" are column references.

Reply:
I see. Then I wonder is there any use case using string args that we can add a test suite for now?